packages/typescript/src/options.ts

```diff
@@ -17,9 +18,16 @@ export function readTsConfig(ts: typeof import('typescript'), tsconfigPath: stri
   if (!existingTsConfig) {
     return {};
   }
+
   const tsconfig = ts.readConfigFile(existingTsConfig, (path) => readFileSync(path, 'utf8'));
 
   if (!tsconfig.config || !tsconfig.config.compilerOptions) return { compilerOptions: {} };
+
+  const extendedTsConfig: string = tsconfig.config.extends;
+  if (tsconfigPath && extendedTsConfig) {
+    tsconfig.config.extends = join(process.cwd(),existingTsConfig, '..', extendedTsConfig);
```

Comment:
Could you tweak `parseCompilerOptions` instead, where the extends path is resolved? 

nit: maybe use `resolve` instead of `join`

Reply:
Actually this bug happens when `tsConfig` option exists at plugin's options so, `parseCompilerOptions` does not relative with plugin's options, it is relative with the config file that read at `readTsConfig`.

Also, you are right because `parseCompilerOptions` should make this. But like I said before, this function doesn't know about `tsConfig` path is given.

I could write a function to make this but it would cause performance issues.

But I did the `resolve` thing 😄.